src/a11y-dialog.ts

```diff
@@ -221,11 +222,12 @@ function $$(selector: string, context: ParentNode = document): HTMLElement[] {
  * Trap the focus inside the given element
  */
 function trapTabKey(el: HTMLElement, event: KeyboardEvent) {
-  const focusableChildren = getFocusableChildren(el)
-  const firstFocusableChild = focusableChildren[0]
-  const lastFocusableChild = focusableChildren.at(-1)
+  const [firstFocusableChild, lastFocusableChild] =
+    getFirstAndLastFocusableChild(el)
 
-  const activeElement = getActiveElement()
+  if (!firstFocusableChild || !lastFocusableChild) return
```

Comment:
What behavior do we expect for a dialog without any focusable elements within it?

Reply:
If there are no focusable elements, `getFirstAndLastFocusableChild()` will return `[null, null]`, so I think I need to cover that case. Something like "if there are no focusable children, do nothing"? because we want to trap the user even if the dialog has no focusable children?